docker-compose.prod.yml

```diff
@@ -46,6 +48,14 @@ services:
       - traefik.enable=true
       - traefik.docker.network=proxy
       - traefik.http.routers.api.rule=Host(`${DOMAIN}`) && Path(`/api`)
+      - logio=yes
+
+  logio:
+    image: geniousphp/logio
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    ports:
+      - 28778:28778
```

Comment:
Can't we expose another port for this, maybe HTTP(S) through Traefik?

Reply:
The issue with exposing it behind Traefik is that we need to create a subdomain for it.
If we append a path (like we did with django), the logio container didn't upload correctly (something similar what happens with the django admin).  Maybe I am configuring wrong Traefik 🤔 

For meanwhile, I think that we can just expose the port in the firewall and later with the domain create a subdomain for it.

Reply:
Sounds good. Can you add a comment?

Reply:
Sure. I will add it.